Makefile: let variables be overridable via environment. (#30)
faf0/sct: README.md

# Installation

## Make-based

> On UNIX-based systems, the most convenient method of building this software is

Reviewer: ```suggestion
On UNIX-based systems, a convenient method of building this software is
```

Author: OK, done.

> using Make. Since the `Makefile` currently is simplistic enough, both the BSD

Reviewer: ```suggestion
using [make(1)](https://www.gnu.org/software/make/). Since the `Makefile` is simple and portable, both the BSD
```

Author: Half NAK on that. I agree with the second part, but the first one is better to stay `Make`. I'm talking about the program family there, not a specific implementation like GNU make. I will, however, add the link to the GNU make part.

Reviewer: Okay, this makes sense.

and GNU make variants will have no problems parsing and executing it
correctly.

The simplest invocation is

~~~sh
make
~~~

which will use the default values for all flags as provided in the Makefile.

Overriding any of the following variables is supported by exporting a variable
with the same name and your desired content to the environment:

  * `CC`
  * `CFLAGS`
  * `LDFLAGS`
  * `PREFIX`
  * `BIN` (the directory into which the resulting binary will be installed)
  * `MAN` (the directory into which the man page will be installed)

>   * `INSTALL` (`install` program used to create directories and copy files)

Reviewer: ```suggestion
  * `INSTALL` (`install(1)` program used to create directories and copy files)
```

Author: Done.

> Both example calls are equivalent and will build the software with a different

Reviewer: ```suggestion
Both example calls are equivalent and will build the software with the specified
```

Author: Done.
